### PR TITLE
Concatenate url for NodeJS 10

### PR DESCRIPTION
**Pull request by mariacarlinahernandez**

### CODE REVIEW

**Comment by d4vsanchez** [comment]
Hey @mariacarlinahernandez template literals works since NodeJS 8 which is already at EOL, this means that template literals do work on NodeJS 10.
What issues are you having and why is this the solution to those issues?

**Review by initdav — COMMENTED** [review]
Hey @mariacarlinahernandez template literals works since NodeJS 4 which is already at EOL, this means that template literals do work on NodeJS 10.
What issues are you having and why is this the solution to those issues?

File: IoT-Projects/SigfoxDevices/Sens'it v2.1/HTTP-SendData/SendData_Ubidots.js [hunk 1]
```diff
@@ -318,7 +318,7 @@ function payloadDecode(data) {
 async function ubidotsRequest(token, label, payload) {
   const options = {
     method: 'POST',
-    url: `https://industrial.api.ubidots.com/api/v1.6/devices/${label}`,
+    url: 'https://industrial.api.ubidots.com/api/v1.6/devices/' + label,
```
Comment (jotathebest) [inline IoT-Projects/SigfoxDevices/Sens'it v2.1/HTTP-SendData/SendData_Ubidots.js:321]: why do you need this change? It creates two string objects and I do not quite see the need to do this

Comment (mariacarlinahernandez) [reply IoT-Projects/SigfoxDevices/Sens'it v2.1/HTTP-SendData/SendData_Ubidots.js:321]: Thanks for the clarification. @d4vsanchez 

I honestly don't know why, as I didn't identify the issue personally. The change was requested by the CS team as I have permission to make changes in the repository.  They reported that after the NodeJS update the example stopped working, then after changing that line of code was working correctly. @jotathebest

**Comment by mariacarlinahernandez** [comment]
An incompatibility was identified after the NodeJS version change. The code must be tested internally.

**Pull request closed without merging**